csv-generator/install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+python -m venv venv
```

Comment:
pipenv will use the virtualenv if it detects one, otherwise create its own. Is there a particular reason why you favour creating a virtualenv here?

Reply:
you don't have to use the install.sh script, it's for my benefit.

Reply:
I was curious what the advantages for you would be. We could also use to build the Dockerfile if it makes it easier.

Reply:
I have this in my .bashrc file:

    alias venv="source venv/bin/activate"

so if a local virtualenv `venv` exists, it just activates it.

Reply:
Okay, might be worth to add it. I have my IDE look for the venv directory by default now so that would work for me as well (I created a link).